.github/workflows/deploy_infrastructure.yaml

```diff
@@ -21,4 +21,14 @@ jobs:
     - name: Install dependencies
       run: npm ci --legacy-peer-deps
     - name: Deploy Dev
-      run: CDK_DEFAULT_ACCOUNT=${{ vars.CDK_DEFAULT_ACCOUNT }} CDK_DEFAULT_REGION=${{ vars.CDK_DEFAULT_REGION }} EMAIL_SENDER=${{ vars.EMAIL_SENDER }} ADMIN_EMAIL=${{ vars.ADMIN_EMAIL }} BASE_DOMAIN=${{ vars.BASE_DOMAIN }} npm run deploy -- --require-approval never
+      env:
+        CDK_DEFAULT_ACCOUNT: ${{ vars.CDK_DEFAULT_ACCOUNT }}
+        CDK_DEFAULT_REGION: ${{ vars.CDK_DEFAULT_REGION }}
+        EMAIL_SENDER: ${{ vars.EMAIL_SENDER }}
+        ADMIN_EMAIL: ${{ vars.ADMIN_EMAIL }}
+        BASE_DOMAIN: ${{ vars.BASE_DOMAIN }}
+        DEV_MONGO_MEMORY: ${{ vars.DEV_MONGO_MEMORY || '2048' }}
```

Comment:
should we be defaulting these since they're also defaulted in the stack file?

Reply:
I was worried about a truthiness issue if I passed vars that are currently undefined, so I made them match instead. In theory no, we don't need the defaults, but if DEV_MONGO_MEMORY ends up being the string 'undefined' or something I want it to work first try. Happy to remove this afterwards and test if you like.